src/prefect/settings.py

```diff
@@ -835,6 +835,12 @@ def default_cloud_ui_url(settings, value):
 )
 """The API's port address (defaults to `4200`)."""
 
+PREFECT_ORION_API_KEEPALIVE = Setting(
+    int,
+    default=5,
```

Comment:
Is this the Uvicorn default value?

Reply:
Yes, based on https://www.uvicorn.org/settings/